airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/streams.py

```diff
@@ -734,18 +755,37 @@ class TicketAudits(SourceZendeskSupportCursorPaginationStream):
 
     transformer = TypeTransformer(TransformConfig.DefaultSchemaNormalization)
 
-    # This endpoint uses a variant of cursor pagination with some differences from cursor pagination used in other endpoints.
-    def request_params(self, next_page_token: Mapping[str, Any] = None, **kwargs) -> MutableMapping[str, Any]:
-        params = {"sort_by": self.cursor_field, "sort_order": "desc", "limit": self.page_size}
 
+class Triggers(SourceZendeskSupportCursorPaginationStream):
+    """Triggers stream: https://developer.zendesk.com/api-reference/ticketing/business-rules/triggers/#list-triggers"""
+
+    # sort_by not supported as request param when doing cursor pagination for this stream (returns DatabaseError)
+    # so in this case incremental syncs for triggers cannot update state at regular intervals
```

Comment:
[Doc](https://developer.zendesk.com/api-reference/ticketing/business-rules/triggers/#list-triggers) says params allow sort_by though? Can it be API version for zendesk account we tested with or something?

Reply:
Not sure. Will see if I can verify this by testing with some other creds

Reply:
Don't see any api versions in documentation apart from v2, v3
`sort_by` param works for offset based pagination but does not work cursor pagination (Maybe they missed updating this in documentation?)
Made a call here to go with cursor pagination at the cost of missing ability to update states regularly because if we go with offset pagination we won't be able to fetch data after 10k records (https://support.zendesk.com/hc/en-us/articles/5591904358938-New-limits-for-offset-based-pagination)

Reply:
Also verified api behaviour with `onefeather` zendesk creds. It is also giving same error

Reply:
Thanks for verifying it more. Resolving this comment.